cmd/atlas/internal/cmdapi/migrate.go

```diff
@@ -232,7 +232,7 @@ func init() {
 	cobra.CheckErr(MigrateApplyCmd.MarkFlagRequired(migrateFlagURL))
 	// Diff flags.
 	urlFlag(&MigrateFlags.DevURL, migrateFlagDevURL, "", MigrateDiffCmd.Flags())
-	urlFlag(&MigrateFlags.ToURL, migrateFlagTo, "", MigrateDiffCmd.Flags())
+	MigrateDiffCmd.Flags().StringSliceVarP(&MigrateFlags.ToURLs, migrateFlagTo, "", nil, "to URLs")
```

Comment:
This changes the help output to be less friendly:
![image](https://user-images.githubusercontent.com/7413593/178736171-344bcc55-47d7-4ef3-8f7d-43c6a081ed5c.png)

Let's do something like this instead?

```
[driver://username:password@address/dbname?param=value ...] select a desired state using the URL format
```